lib/mp4box-player/src/Mp4boxPlayer.js

```diff
@@ -733,7 +746,7 @@ export class Mp4boxPlayer {
         sb.ms = ms;
         sb.id = track_id;
         this.mp4boxfile.setSegmentOptions(track_id, sb, {
-          nbSamples: this.config.segmentSize,
+          nbSamples: this.videoFps * (this.CHUNK_DURATION/2)
```

Comment:
Could you explain how you determined that a `CHUNK_DURATION` of 3 seconds is optimal? Additionally, could you provide more details about what `CHUNK_DURATION` represents exactly?

I'm also interested in understanding the logic behind the `nbSamples` formula you used. I mean, How did you arrive at this formula FPS * CHUNK_DURATION/2

<img width="971" alt="image" src="https://github.com/stacknide/mp4box-apps/assets/60546840/a511ccfc-743c-409a-815a-8249df914034">

Reply:
CHUNK_DURATION is got from testing. Usually for fast start,, we can try from 0.5 -> 2 seconds for live streaming . Something used in LowLatency-HLS optiimization for live. But since ours is VOD but need a quick start, i tried from 2, to upwards of 5seconds. 

but instead of creating segments for entire chunk_size, we are trying to do CHUNK_DURATION/2 for each segment, so the player can load the first playable segment with low memory footprint and slightly faster. 

nbSample = number of frames  => CHUNK_DURATION/2 * videoFPS just represents this duration in FPS number.

Reply:
if we use CHUNK_DURATION = 2 -> there seems to be buffering under 100mpbs conditions for 4k. 
4,5 sees some stability but more start time, 3s seems to be to be optimal value as per my tests. so the value

Reply:
What I found out by testing was: having a 4MB `chunkSize` & a lower `nb_samples` (segment Size config field) helped in overcoming our current issue with playing high resolution videos
  
---------------

Here's what I understood right now:

`nb_samples` is the number of samples. In case of video tracks, "samples" mean "frames"
  - so `nb_samples` means "number of frames per segment"
  - A segment is a fragmented video chunk. Every segment's duration might vary in case of adaptive bitrate streaming, but in our use case, we will always have a constant bitrate. So Segment duration will always be a constant. Also, every segment have the same FPS
  - so a better way of writing the formula of calculating `nb_samples` would be: 
  ```
  nb_samples = FPS * segmentDurationInSecs
             = number of frames present in a segment of duration segmentDurationInSecs
  ```


If I am not wrong with my understanding of what `nb_samples` means, perhaps we must remove `CHUNK_DURATION/2` factor and use `segmentDurationInSecs` factor instead. What do you think?